www/src/pages/en/deployment/index.astro

```diff
@@ -0,0 +1,24 @@
+---
+import Layout from "../../../layouts/docs.astro";
+import IndexPage from "../../../components/docs/indexPage.astro";
+import { Frontmatter, SIDEBAR } from "../../../config";
+import { getLanguageFromURL } from "../../../languages";
+
+const frontmatter: Frontmatter = {
+  title: "Deployment",
```

Comment:
how will this work with translation? could we stick the frontmatter in markdown files for each language with a fallback to english, or something like that?

Reply:
This file will be in all langs so when they copy the `en` folder they'll just have to translate this too. Not a whole lot